### PR TITLE
[JB][SUPP-464] filter out any instalments that are less than £1 from …

### PR DESCRIPTION
**Pull request by barnesjake**
…being displayed on the instalments screen

### CODE REVIEW

File: test/controllers/InstalmentsControllerSpec.scala [hunk 1]
```diff
@@ -130,4 +133,19 @@ class InstalmentsControllerSpec extends ItSpec {
...
   }
 
+  "InstalmentsController.retrieveInstalmentOptions" - {
+    "should filter out any instalments that are less than £1" in {
+      val plansIncludingOneLessThanAPound: List[PaymentPlan] = List(
+        TdAll.paymentPlan(1, amountDue = AmountDue(AmountInPence(1))),
+        TdAll.paymentPlan(2, amountDue = AmountDue(AmountInPence(99))),
+        TdAll.paymentPlan(3),
+        TdAll.paymentPlan(4),
+        TdAll.paymentPlan(5),
```
Comment (darrell-plant) [inline test/controllers/InstalmentsControllerSpec.scala:143]: maybe clearer to make the `amountDue` values explicit here

**Review by darrell-plant — DISMISSED** [review]

**Comment by darrell-plant** [comment]
So, is it okay just to filter out the smaller amounts? Does that not introduce some inaccuracy smewhere?

File: test/controllers/InstalmentsControllerSpec.scala (hunk 1 above)
Comment (barnesjake) [reply test/controllers/InstalmentsControllerSpec.scala:143]: Sure

**Comment by barnesjake** [comment]
This is only on the displaying of the instalment options. In theory if ttp only returned options less than £1 it would be an empty list, but I don't think this is possible due to validation prior to this and also ttp rules (something to do with minimum debt amount in the eligibility checks)

**Review by darrell-plant — APPROVED** [review]

**Pull request merged**